packages/synapse-react-client/src/utils/functions/sqlFunctions.test.ts

```diff
@@ -66,12 +69,12 @@ describe('getAdditionalFilters', () => {
     }
     const operator: SQLOperator = ColumnSingleValueFilterOperator.LIKE
     // if no search params are there, then it should return undefined
-    expect(getAdditionalFilters('syn16858331', undefined, operator)).toBe(
+    expect(getAdditionalFilters(undefined, operator, 'syn16858331')).toBe(
```

Comment:
reordered parameters for this utility function. The 3rd parameter was least likely to be used (the sessionStorage key in which to store filters)